eox_tenant/edxapp_wrapper/backends/certificates_module_i_v1.py

```diff
@@ -2,13 +2,17 @@
 Backend Certificates file, here should be all the necessary methods,
 classes and modules from lms.djangoapss.certificates.
 """
+from django.conf import settings
+
 from lms.djangoapps.certificates import models  # pylint: disable=import-error
 
+from eox_tenant.test_utils import TestCertificatesModels
+
 
 def get_certificates_models():
     """
     Backend function.
 
     Return <lms.djangoapps.certificates.models>.
     """
-    return models
+    return models if getattr(settings, 'USE_EOX_TENANT', False) else TestCertificatesModels()
```

Comment:
Is this line present only for the tests on the platform?

I find the logic here difficult to follow.
In the cases where we are testing eox-tenant only, we should have USE_EOX_TENANT set to true, but we will not use this backend. Is that correct?

Could you at least  make a comment explaining the necessity for this?